google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -80,6 +80,9 @@
   /** Fragment component or {@code null} for none. */
   private String fragment;
 
+  /** When true the URL is used `as is` (without encoding, decoding and escaping)
```

Comment:
Sorry, the Javadoc is still not terminated, and this is causing the build to fail.

```suggestion
  /**
    * If true, the URL string originally given is used as is (without encoding, decoding and
    * escaping) whenever referenced; otherwise, part of the URL string may be encoded or decoded as
    * deemed appropriate or necessary.
    */
```